essentials/src/main/java/Lab3.java

```diff
@@ -37,17 +41,17 @@ public static void main (String[] args) {
     private static Pipeline buildPipeline() {
         Pipeline p = Pipeline.create();
 
-        // 1 - Read from the Trade Source (sources.TradeSource)
+        p.drawFrom(TradeSource.tradeSource(1))
+         .withNativeTimestamps(0 )
 
-        // 2 - Without timestamps
+         // Detect if price between two consecutive trades drops by more then 200
 
-        // 3 - Compute max price in a rolling way
-        // - the max will be updated and emitted with each incoming trade
-        // - use com.hazelcast.jet.aggregate.AggregateOperations library with aggregations
-        // - use com.hazelcast.jet.function.ComparatorEx library
+         // Use the mapStateful to keep price of previous Trade
+         // - Consider using com.hazelcast.util.MutableLong as a mutable container for long values
```

Comment:
LongAccumulator is better fit

Reply:
I don't need to accumulate, just keep latest longs value in a mutable object. What's the benefit of using LongAccumulator?

Reply:
MutableLong API has very poor documentation. LongAccumulator is nicer, also has a set method, you don't necessarily need to accumulate and fits more uses case and also directly part of Jet API. MutableLong may even get removed in 4.0.